styles/shared/positions.css

```diff
@@ -0,0 +1,94 @@
+/*********************************************************************************************************/
+/****************************************** Position Styles *********************************************/
+
+.pos-rel {
+    position: relative;
+}
+
+.pos-abs {
+    position: absolute;
+}
+
+.top-rgt-5 {
+    top: 5%;
+    right: 5%;
+}
+
+.top-lft-5 {
+    top: 5%;
+    left: 5%;
+}
+
+.btm-rgt-5 {
+    bottom: 5%;
+    right: 5%;
+}
+
+.btm-lft-5 {
+    bottom: 5%;
+    left: 5%;
+}
+
+.hw-33 {
+    height: 33.33%;
+    width: 33.33%;
+}
+
+.rot-c-45 {
+    transform: rotate(45deg);
+}
+
+/*********************************************************************************************************/
+/*********************************************************************************************************/
+
+/*********************************************************************************************************/
+/********************************************* Display Styles ****************************************************/
+
+.hide-rs {
+    visibility: hidden;
+}
+
+.hide-drs {
+    display: none;
+}
+
+.show {
+    display: block;
+}
+
+.o-00 {
+    opacity: 0;
+}
+.o-01 {
+    opacity: 0.1;
+}
+.o-02 {
+    opacity: 0.2;
+}
+.o-03 {
+    opacity: 0.3;
+}
+.o-04 {
+    opacity: 0.4;
+}
+.o-05 {
+    opacity: 0.5;
+}
```

Comment:
Create classes only when you are repeating yourself.
If you are using these classes at multiple places, then it's okay.